types/meson.build

```diff
@@ -66,3 +66,11 @@ wlr_files += files(
 	'wlr_xdg_decoration_v1.c',
 	'wlr_xdg_output_v1.c',
 )
+if uuid.found() or uuid_create
+	conf_data.set10('WLR_HAS_XDG_FOREIGN', true)
```

Comment:
Can we move this one line to the main `meson.build`? It's too easy to incorrectly include another Meson file that depends on this being set before executing `subdir('types')`.